sdks/python/apache_beam/transforms/external.py

```diff
@@ -670,7 +670,8 @@ def __enter__(self):
       self._service_provider = subprocess_server.JavaJarServer(
           ExpansionAndArtifactRetrievalStub,
           self._path_to_jar,
-          self._extra_args)
+          self._extra_args + [f'--filesToStage={self._path_to_jar}'],
```

Comment:
The behavior is a little different here. We set `--filesToStage` even if extra_args was not None in `__init__`. I'm not sure if the previous behavior was intentional or not.

Reply:
I would keep the same behavior.

cc: @robertwb

Reply:
nit: remove extra ","

Reply:
Let's keep the same behavior, otherwise a user has no way of not specifying this argument (if, say, they have an expansion service not based on our jar).

Reply:
(This is what I did in https://github.com/apache/beam/pull/15891 . I still think it'd be good to get this in for the release, but if it's the only change that we'd need to CP I'll defer to the release manager as whether it meets the bar.)

Reply:
Sounds good. LGTMed https://github.com/apache/beam/pull/15891. To be clear I think this is a true regression/blocker. IIUC this will break x-lang wrappers in production.